src/Util/FormBuilderIterator.php

```diff
@@ -39,7 +39,7 @@ public function __construct(FormBuilderInterface $formBuilder, ?string $prefix =
     {
         parent::__construct();
         $this->formBuilder = $formBuilder;
-        $this->prefix = $prefix ?? $formBuilder->getName();
+        $this->prefix = $prefix ?: $formBuilder->getName();
```

Comment:
I'm wondering if `??` is not better than `?:` in this case...

Reply:
I guess not:

```php
$this->prefix = $prefix ?? $formBuilder->getName(); // will always return $prefix ($prefix is not null)
$this->prefix = $prefix ?: $formBuilder->getName(); // will return $formBuilder->getName(); in case $prefix is false
```

Reply:
$prefix is null or a string.

If someone pass `''`, shouldn't we use `''` as a prefix instead of overriding the prefix to $formBuilder->getName() ?
IMHO we should only override the prefix if null is passed. That's the job of `??`

Reply:
My comment was based on the phpdoc:
```php
/**
 * @param bool $prefix
 */
public function __construct(FormBuilderInterface $formBuilder, $prefix = false)
{
```
I haven't check the usages to see if it can be a `string`, if so, the phpdoc should be updated.

Reply:
We moved from https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Util/FormBuilderIterator.php#L53
to https://github.com/sonata-project/SonataAdminBundle/blob/master/src/Util/FormBuilderIterator.php#L38

So `??` is better in master

Reply:
Looking at the usages, the only place where the `$prefix` was passed is:

https://github.com/sonata-project/SonataAdminBundle/blob/2731298ece3cccd21a93f8ae22d11d509228357e/src/Util/FormBuilderIterator.php#L90

In `master` that argument was removed in https://github.com/sonata-project/SonataAdminBundle/pull/6533, I don't know what was the use of this, but looks like it should have been deprecated in `3.x` and removed in `master`.